### DISCUSSION TITLE
Problem with Swift env.step(dt) in the new version

### DISCUSSION THREAD

**Discussion by waelbachta** [petercorke/robotics-toolbox-python#341] [Q&A]
Hi,

I am running the toolbox using an ubuntu 20.04 environment. When launching an example with swift, I get the following error :

File "demo.py", line 60, in <module>
    env.step(0.000001)
  File "/home/wael/TP_NEW/venv/lib/python3.8/site-packages/swift/Swift.py", line 209, in step
    obj._propogate_scene_tree()
AttributeError: 'NoneType' object has no attribute '_propogate_scene_tree'

Thank you for your help !

Wael.

**Comment by DevilCJS89** [comment]
The key words: AttributeError: 'NoneType' object has no attribute '_propogate_scene_tree' 
you should chek your packages use"pip list "or "conda list"
rename the wrong file name 
run again
if there are any question , I'll try my best to help you resolve it.
